Objects/stringlib/unicode_format.h

```diff
@@ -989,6 +989,24 @@ formatteriter_dealloc(formatteriterobject *it)
    format_spec is the string after the ':'.  mibht be None
    conversion is either None, or the string after the '!'
 */
+
+static PyStructSequence_Field formatter_iter_result_fields[] = {
+    {"literal_text", "Span of literal text."},
+    {"field_name", "Specifies the object whose value is to be formatted."},
+    {"format_spec", "Contains a specification of how the value should be presented."},
+    {"conversion", "The conversion to be used. One of: ‘s’ (str), ‘r’ (repr) and ‘a’ (ascii)."},
+    {NULL}
+};
+
+static PyTypeObject FormatterIterResultType;
+
+static PyStructSequence_Desc formatter_iter_result_desc = {
+    "FormatterItem",
```

Comment:
Shouldn't a module name be added?

Reply:
Do you mean to expose the type in a module or to add a `doc` to the second field?

Reply:
Shouldn't the name of the type include the name of the module?

Reply:
Done in 3ee0266.